compact/ecal/backward_PbWO4.xml

```diff
@@ -39,11 +49,135 @@
             r12max="EcalEndcapN_structure_ring_max"
             SFlength="EcalEndcapN_structure_ring_length"
             CMlength="EcalEndcapN_CrystalModule_length + 2.*EcalEndcapN_CrystalModule_wrap_thickness"
+            NEEMC_PR="EcalEndcapN_Protate"
+            NEEMC_NR="EcalEndcapN_Nrotate"
+            NEEMC_OR_RS="EcalEndcapN_Oring_relative_shift"
+            Inner_a="EcalEndcapN_Iring_axisA"
+            Inner_b="EcalEndcapN_Iring_axisB"
             envelope="true"
-            vis_struc="RPVis"
-            vis_steel_gap="AnlGray"
             ring_material="StainlessSteel"
-            sector="1">
+            sector="1"
+            vis_struc="RPVis"
+            vis_steel_gap="AnlGray">
+
+          <comment>
+            The position information[cm] of inner supporting structure [extrudedpolygon shape, 12 points in total]
+          </comment>
+          <positions name="extrudedpolygon_pos">
+            <position x="6.15"  y="6.15"/>
+            <position x="6.15"  y="4.1"/>
+            <position x="8.2"   y="4.1"/>
+            <position x="8.2"   y="-4.1"/>
+            <position x="6.15"  y="-4.1"/>
+            <position x="6.15"  y="-6.15"/>
+            <position x="-6.15" y="-6.15"/>
+            <position x="-6.15" y="-4.1"/>
+            <position x="-8.2"  y="-4.1"/>
+            <position x="-8.2"  y="4.1"/>
+            <position x="-6.15" y="4.1"/>
+            <position x="-6.15" y="6.15"/>
+          </positions>
+
+          <comment>
+            The position information[cm] of additional modules to fill the gap
+            between the sensitive region and inner/outer frame structure. [92 posints in total]
+          </comment>
+          <positions name="additionalmodule_pos">
+            <position x="1.025"  y="7.175"/>
```

Comment:
Same here, all positions should have an explicit unit multiplied.

Reply:
done